internal/compiler/typeloader.rs

```diff
@@ -988,11 +1008,54 @@ impl TypeLoader {
                 imports.push(import);
                 continue;
             }
+
             dependencies_futures.push(Box::pin(async move {
-                let file = import.file.as_str();
+                let import_file = import.file.clone();
+                if let Some(maybe_library_import) = import_file.strip_prefix('@') {
+                    if let Some(library_name) = std::env::var(format!(
+                        "DEP_{}_SLINT_LIBRARY_NAME",
+                        maybe_library_import.to_uppercase()
+                    ))
+                    .ok()
```

Comment:
I am not sure if this (the type loader) is the right location to read the environment variables.
Perhaps they should be fed in the CompilerConfig somehow.

But this can always be changed later, so i think we can accept that now.